src/java.base/share/classes/java/security/package-info.java

```diff
@@ -79,6 +79,7 @@
  *
  * </ul>
  *
+ * @spec security/standard-names.html Java Security Standard Algorithm Names
```

Comment:
How about the other 2 under `Package Specification` (from line 49-53)? Does this PR need to cover them also?

Reply:
Same question applies for the java/security/interfaces/package-info.java file as well.